lib/server/routes/buckets.js

```diff
@@ -187,37 +187,43 @@ BucketsRouter.prototype.getBucketById = function (req, res, next) {
  * @param {http.ServerResponse} res
  * @param {Function} next
  */
-BucketsRouter.prototype.createBucket = function (req, res, next) {
-  const Bucket = this.storage.models.Bucket;
+BucketsRouter.prototype.createBucket = async function (req, res, next) {
+  const { body, user, pubkey } = req;
```

Comment:
Be careful with destructuring if you are not enforcing the fields explicitly, an unhandled exception could arise.